eahub/profiles/views.py

```diff
@@ -162,4 +163,9 @@ def delete_profile(request: HttpRequest) -> HttpResponse:
 
 
 def profiles(request) -> HttpResponse:
-    return render(request, "eahub/profiles.html", {"feedback_form": FeedbackForm()})
+    approved_user = False
+    if request.user.is_authenticated:
+        profiles = Profile.objects.all().filter(user=request.user)
+        if len(profiles) == 1 and profiles[0].is_approved:
+            approved_user = True
+    return render(request, "eahub/profiles.html", {"feedback_form": FeedbackForm(), "approved_user": approved_user })
```

Comment:
could this be replaced with `{{ request.user.profile.is_approved }}` in html? looks like it could save few lines of python then, since django template fallbacks to render an empty string?

Reply:
done